src/components/attendee/VideoComms/scss/VideoCommsParticipant.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/constants";
+
+.videoCommsParticipant {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  height: 100%;
+  max-width: $video-participant-optimal-width;
+  min-width: $video-participant-optimal-width;
+  max-height: $video-participant-optimal-height;
```

Comment:
These would look more like `$border-radius--xl` if they're renamed into `$video-participant-width--optimal` and `$video-participant-height--optimal`